packages/backend/src/services/SavedSqlService/SavedSqlService.ts

```diff
@@ -412,23 +412,41 @@ export class SavedSqlService extends BaseService {
         body: SqlRunnerPivotQueryBody,
         context?: QueryExecutionContext,
     ): Promise<{ jobId: string }> {
+        const { savedSqlUuid } = body;
```

Comment:
there might be just the `slug` on view mode btw, could we account for that as well?

Reply:
or is it handled by the FE?

Reply:
I see it now, ignore!